src/vs/workbench/services/editor/test/browser/editorService.test.ts

```diff
@@ -1703,13 +1703,10 @@ suite('EditorService', () => {
 		assert.strictEqual(part.activeGroup, sideGroup);
 		assert.notStrictEqual(rootGroup, sideGroup);
 
-		part.arrangeGroups(GroupsArrangement.MAXIMIZE, part.activeGroup);
+		part.arrangeGroups(GroupsArrangement.EXPAND, part.activeGroup);
 
 		await rootGroup.closeEditor(input2);
 		assert.strictEqual(part.activeGroup, sideGroup);
-
```

Comment:
Any reason we cannot test this anymore here?

Reply:
I removed the API to access it. `isGroupMaximized` and `isGroupExpanded` are both private methods as they are not used from the outside.

Reply:
I think I had the methods public specifically to test this here. I would bring it back, so that the behaviour is asserted as before.